scripts/nitro-testnode.sh

```diff
@@ -46,7 +46,7 @@ then
   # `release` branch.
   git checkout 8cb6b84e31909157d431e7e4af9fb83799443e00 || exit
 
-  ./test-node.bash --no-run --init --no-tokenbridge || exit
+  ./test-node.bash --no-run --init || exit
```

Comment:
Why do we need `tokenbridge`?

Reply:
In this version of the test node `--no-tokenbridge` behaviour is by default.
Before it was different

Reply:
Fair enough, thanks!